pages.es/common/pio-project.md

```diff
@@ -0,0 +1,24 @@
+# pio project
+
+> Administra proyectos PlatformIO.
+> Más información: <https://docs.platformio.org/en/latest/core/userguide/project/>.
+
+- Inicializa un nuevo proyecto PlatformIO:
+
+`pio project init`
+
+- Inicializa un nuevo proyecto PlatformIO en un directorio específico:
+
+`pio project init --project-dir {{ruta/al/directorio_del_proyecto}}`
+
+- Inicializa un nuevo proyecto PlatformIO, especificando un ID de la board:
```

Comment:
```suggestion
- Inicializa un nuevo proyecto PlatformIO, especificando un ID del board:
```

Reply:
Seems that the commit is already been applied, I'm ok with that. @kant don't hesitate to approve and merge this one.